docs/overrides/home.html

```diff
@@ -0,0 +1,151 @@
+			<h1>Take handwritten notes with ease.</h1>
+			<h2>From solving a physics problem to making travel diaries.</h2>
+		</div>
+		<p>Xournal++ is an open-source and cross-platform note-taking software that is fast, flexible, and functional. A modern rewrite and a more feature-rich version of the wonderful <span><a class ="Xournal-link" href="https://sourceforge.net/projects/xournal/">Xournal</a></span> program.</p>
+		<img src="img/Physics Problem.png">
+		<div class="download-hyperlink upper-button">
+			<a href="{{ 'installation/' | url }}">Download Xournal++</a>
+		</div>
+	</div>
+	
+	<div class="feature-list">
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Handwrite your notes</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+		        		<p class="bullet-point">|</p><p>Support for pressure-sensitive stylus and drawing tablets (Wacom, Huion, XP-Pen, etc.)</p>
+		        	</div>
+		        </div>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+						<p class="bullet-point">|</p><p>Robust and customizable pen, highlighters, and eraser tool, allowing you to write how you want to write</p>
+					</div>
+				</div>
+				<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Handwritten.png">
+		</div>
+		
+		<div class="feature feature-left">
+		    <div class="feature-text">
+		        <h2>Keep your work organized</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+		        		<p class="bullet-point">|</p><p>Use layers to make complex notes that are still pleasant to work with</p>
+		        	</div>
+		        </div>
+		        <div class="feature-description-container">
+		        	<div class="feature-description last-description">
+						<p class="bullet-point">|</p><p>The page previews and PDF bookmarks allow for easy navigation of your notes</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src = "/img/Layers.gif">
+		</div>
+		
+		<div class="feature feature-right">
+		    <div class="feature-text">
+		        <h2>Enhance and accelerate your note-taking</h2>
+		        <div class="feature-description-container">
+		        	<div class="feature-description">
+						<p class="bullet-point">|</p><p>Add images and create various shapes, from circles to splines to axis</p>
+					</div>
+				</div>
+				<div class="feature-description-container">
+					<div class="feature-description last-description">
+	        			<p class="bullet-point">|</p><p>Snap objects to rectangular grid or degrees of rotation</p>
+	        		</div>
+	        	</div>
+	        	<div class="feature-link">
+					<a href="#">Learn more</a>
+				</div>
+		    </div>
+		    <img src="/img/Create Shape.png">
```

Comment:
This image does not really represent the feature. Instead, use a GIF that actually shows the process of drawing a shape.

Reply:
I beg to differ. I think it's quite clear that the objects in there are drawn are drawn with the tools in the apps. Moreover, this image shows how users may utilize the app's many shape creation tools in actually aiding their note creation process. Which isn't the case with a GIF of merely drawing a circle.

Reply:
> I beg to differ. I think it's quite clear that the objects in there are drawn are drawn with the tools in the apps. Moreover, this image shows how users may utilize the app's many shape creation tools in actually aiding their note creation process. Which isn't the case with a GIF of merely drawing a circle.

Yes but only partially. The shape recognizer is missing completely and most of the shapes can actually be drawn by the line tool or free hand (Spline). It is not really obvious that the spline was created using the tool. We are talking about a user that has no idea what features Xournal++ has. So the most obvious idea would be, that the spline was drawn freely, not using a specific tool.

> Snap objects to rectangular grid or degrees of rotation

This is not shown by the current image at all.

Reply:
> > I beg to differ. I think it's quite clear that the objects in there are drawn are drawn with the tools in the apps. Moreover, this image shows how users may utilize the app's many shape creation tools in actually aiding their note creation process. Which isn't the case with a GIF of merely drawing a circle.
> 
> Yes but only partially. The shape recognizer is missing completely and most of the shapes can actually be drawn by the line tool or free hand (Spline). It is not really obvious that the spline was created using the tool. We are talking about a user that has no idea what features Xournal++ has. So the most obvious idea would be, that the spline was drawn freely, not using a specific tool.

Alright fair enough. But if I'm going to make a GIF of creating shapes, then it's just not possible to showcase every single shape creation tool. It would take too long, speeding it up would make visitors barely know what they're seeing. If that's a fine tradeoff then I'll go make it.

> > Snap objects to rectangular grid or degrees of rotation
> 
> This is not shown by the current image at all.
Well not all of the feature in the description can be shown. If you scroll up to the previous feature showcase, the page previews that is mentioned there isn't shown in the GIF.

Reply:
> Alright fair enough. But if I'm going to make a GIF of creating shapes, then it's just not possible to showcase every single shape creation tool. It would take too long, speeding it up would make visitors barely know what they're seeing. If that's a fine tradeoff then I'll go make it.

Then you should focus on the most impressive ones. A coordinate system with a spline for sure and a circle somewhere, I'd propose.

> Well not all of the feature in the description can be shown. If you scroll up to the previous feature showcase, the page previews that is mentioned there isn't shown in the GIF.

Valid point.

Reply:
I think the current image is fine. Any demonstration that requires a longer GIF should go into a "trailer" or "feature" video instead. We only need to showcase the basic capabilities here.